internal/app/machined/pkg/runtime/logging/sender_jsonlines.go

```diff
@@ -55,6 +59,10 @@ func (j *jsonLinesSender) marshalJSON(e *runtime.LogEvent) ([]byte, error) {
 	m["talos-time"] = e.Time.Format(time.RFC3339Nano)
 	m["talos-level"] = e.Level.String()
 
+	for k, v := range j.extraTags {
```

Comment:
should  we validate the standard ones are not overriden?

Reply:
I don't think so... if someone wants it... they get it?